opengemini/query.go

```diff
@@ -41,12 +41,8 @@ func (c *client) Query(q Query) (*QueryResult, error) {
 	req.queryValues.Add("rp", q.RetentionPolicy)
 	req.queryValues.Add("epoch", q.Precision.Epoch())
 
-	//Encoding
-	if c.config.Encoding == MSGPACK {
-		if req.header == nil {
-			req.header = make(http.Header)
-		}
-		req.header.Set("Accept", "application/x-msgpack")
+	if c.config.Codec == MSGPACK {
```

Comment:
if `Codec` is not set, the default value `MSGPACK` is used, is this the case?

Reply:
Fixed